addons/account/views/account_payment_view.xml

```diff
@@ -276,7 +276,8 @@
                                             'readonly': [('state', '!=', 'draft')],
                                         }"/>
 
-                                <field name="partner_bank_id" context="{'default_partner_id': partner_id}" string="Vendor Bank Account"
+                                <field name="partner_bank_id" string="Vendor Bank Account"
+                                        context="{'default_partner_id': partner_id, 'default_allow_out_payment': True}"
```

Comment:
This context does not seem to work (neither allow_out_payment nor partner_id are affected). @oco-odoo